substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/doc-files/SerializationConfigurationFilesHelp.txt

```diff
@@ -1,7 +1,8 @@
 One or several (comma-separated) paths to JSON files that specify lists of serialization configurations.
+
 The structure is described in the following schema:
 
-  https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/assets/serialization-config-schema-v1.0.0.json
+  https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/assets/serialization-config-schema-v1.1.0.json
```

Comment:
```suggestion
  https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/assets/serialization-config-schema-v1.0.0.json
```

Same as https://github.com/graalvm/graalvm-community-jdk21u/pull/23/files#r1775658358 plus `proxies` are not supported in legacy config.